### DISCUSSION TITLE
Can you manually change the filter state

### DISCUSSION THREAD

**Discussion by lucianthorr** [charmbracelet/bubbles#610] [Q&A]
Hi.  I'm trying to create my first bubbletea app that acts as a variation on the list.Model.
I'd like it such that the filter is always enabled so that the user typing filters the dropdown and then up and down arrows allow the user to leave the filter's textInput and select an item.
Is there a way to manually set the FilterState so that it is always Filtering when users send key messages?  Or is there another way to implement that behavior that I'm overlooking?
Thanks

**Comment by blvrd** [comment]
#335 adds a `SetFilterState` method that should do the trick. It's not merged, but it was recently approved.

**Reply by meowgorithm** [reply]
Merged! https://github.com/charmbracelet/bubbles/pull/335

**Reply by lucianthorr** [reply]
Thanks!
